### DISCUSSION TITLE
Module available in US?

### DISCUSSION THREAD

**Discussion by eigenphase** [jomjol/AI-on-the-edge-device#1224] [Q&A]
The hardware page scares me. I am wondering if anyone from the US can give me a tip which module worked.
The HW page only lists https://arduino-projekte.info/produkt/esp32-cam-v2-integriertem-ch340-mit-ov2640-kamera-modul/ which is Germany only.

Could this one work? https://www.amazon.com/HiLetgo-ESP32-CAM-Development-Bluetooth-Raspberry/dp/B07RXPHYNM

**Comment by DanielWe2** [comment]
I think most ESP32 CAM modules should work.  Most importantly: It needs to have 4 MB of RAM (or more if they exist?).

The biggest issue is poor quality of some of the boards (for example power regulators with not enough amps). But in general most boards should be fine.

The one linked should be OK and includes a board to directly flash it.

**Reply by eigenphase** [reply]
Awesome! Ordered mine and will report back.
